src/frontend/components/UI/ControllerHints/index.tsx

```diff
@@ -121,7 +121,7 @@ export default function ControllerHints() {
 
   useEffect(() => {
     // set the brand for the images to use
-    if (activeController.match(/sony|0ce6|PS3|PLAYSTATION|0268|'2563.*0523/i)) {
```

Comment:
maybe we can add 054c instead of replacing it? I don't remember why I put 0ce6 at some point there but maybe removing it breaks this for some users

Reply:
the 0ce6 is specific to DS5 controller, I think it's save to delete when we have a vendor id instead

Reply:
yeah a vendor ID is probably the best idea